src/test/java/io/aiven/kafka/connect/s3/testutils/BucketAccessor.java

```diff
@@ -50,6 +51,13 @@ public final void createBucket() {
         s3.createBucket(bucketName);
     }
 
+    public final void removeBucket() {
+        for (final S3ObjectSummary s3Obj : s3.listObjects(bucketName).getObjectSummaries()) {
+            s3.deleteObject(bucketName, s3Obj.getKey());
```

Comment:
S3 client support bulk delete please take a look here: 
https://github.com/aiven/aiven-elasticsearch-repositories/blob/b3d1e139ad69912d528b4d38212e5640b04f9d2b/repository-s3/src/main/java/io/aiven/elasticsearch/repositories/s3/S3RepositoryStorageIOProvider.java#L174

Reply:
Bulk limit in `10_000` is not our case I suppose

Reply:
Nope it is less than 10000). Good stuff